.github/config.yml

```diff
@@ -38,7 +38,7 @@ firstPRMergeComment: |
  <p>People often get stuck at the same steps, so you might be able to help someone get unstuck, or help lead them to some documentation that'd help. Reach out and be encouraging and friendly! 😄 🎉 </p>
  <p>Read about <a href="https://github.com/publiclab/plots2/issues/3458">how to help support another newcomer here</a>, or find <a href="https://github.com/publiclab/plots2/labels/support">other ways to offer mutual support here</a>.</p>
  </details>
-<p>Join our Gitter channel <a href="https://gitter.im/publiclab/publiclab">here</a> to connect with other contributors</p> :inbox_tray:
-
+ <br />
+ You can also join our [Gitter channel](https://gitter.im/publiclab/publiclab) to connect with other contributors :inbox_tray:
```

Comment:
Ooh looks like the Github config file doesn't allow this syntax for links. Can you change line 42 to this?
```
 You can also join our <a href="https://gitter.im/publiclab/publiclab">Gitter channel</a> to connect with other contributors :inbox_tray:
```

You can open up a follow-up PR for that if you're interested. Thanks!!

Reply:
You can also remove line 41(`<br />`). Thanks!!

Reply:
Thank you @TildaDares for helping to debug this ❤️

I'm curious though as to why the markdown syntax failed to work for lines 41 - 42.

In the same file ([on lines 21 - 22](https://github.com/publiclab/plots2/blob/a55a5869650df75194005b259200c80d77f07924/.github/config.yml#L21)), we have roughly the same syntax: a line break followed by the regular markdown syntax for links, and it's displaying just fine 🤔 

Do you have any idea why it seems to work for some lines and not for others? Thank you!